packages/cli/src/commands/start.ts

```diff
@@ -1,8 +1,13 @@
-import {Command, flags} from "@oclif/command"
 import {dev, prod} from "@blitzjs/server"
-
+import {Command, flags} from "@oclif/command"
+import fs from "fs"
+import path from "path"
+import pkgDir from "pkg-dir"
 import {runPrismaGeneration} from "./db"
 
+const projectRoot = pkgDir.sync() || process.cwd()
+const isTypescript = fs.existsSync(path.join(projectRoot, "tsconfig.json"))
```

Comment:
This is probably fine for now but we should look at how this sort of config stuff is managed in a different PR.

Reply:
What is your idea here? Should we create new discussion thread about config stuff here on GH?

Reply:
It would be good not to have to drill config down so many levels deep. Ideally there is a thing that determines information about the environment that is always run. We kind of have a place that might be more natural for this stuff perhaps [here](https://github.com/blitz-js/blitz/blob/canary/packages/server/src/config.ts) rather than have to remember to run that check everytime.

Reply:
I think I get your point. Typescript checking in cli level really seems unnecessary. It should be moved to `server/config.ts`. 

I opened new discussion about drilling config down here where I am also including ts checking move: https://github.com/blitz-js/blitz/discussions/941 so we can continue in discussion there.